app/src/components/main/breadcrumbs/breadcrumb/breadcrumb.scss

```diff
@@ -5,6 +5,8 @@
   padding-right: 5px;
   font-size: 12px;
   color: $COLOR--charcoal-grey;
+  word-break: break-word;
```

Comment:
This value doesn't work in firefox. Please add fallback `break-all`:
```
word-break break-all;

@supports(word-break: break-word) {
  word-break: break-word;
}
```

Reply:
You right! Thanks!